hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/OzoneClientConfig.java

```diff
@@ -124,6 +124,13 @@
       tags = ConfigTag.CLIENT)
   private boolean checksumVerify = true;
 
+  @Config(key = "max.ec.stripe.write.retries.on.failures",
```

Comment:
I would suggest to either use max.retries for this purpose, or if the scope of that config is to large and it is better to have a separate config, name it similarly as for example max.ec.retries, if it makes sense to distinguish between write and read here then max.ec.write.retries, and only if we also want to distinguish between stripe write retries and ec key write retries then probably max.ec.stripe.write.retries. What do you think? I would definitely remove at least the ".on.failures" part as it is fairly obvious that retries will happen if there was a failure.

Also I think it should default to the value of max.retries instead of an arbitrary number.

Reply:
I wanted to separate this from general write retries. In this case, specifically we do retries at stripe boundary and if we can write that stripe after trying this configured times we will give up the write completely. Where as the regular write.retries(Mostly in non EC case) are per block level and after it tried that many times,I think it would just give up that block write and move on with new block allocations. So, I feel we can keep little high number to make sure we will not give up too early. Based on the practice, we may have to introduce incremental waits between retries to slow down that unnecessary block allocations from OM.

Reply:
Ok, I understand. One small thing, the constant name in OzoneConfigKeys is still having the _ON_FAILURE suffix, is that intentional?

Reply:
Thanks for catching it. It was just overlook and missed it. Just fixed it. Thanks